### PR TITLE
Feature/busify expressions

### PR DESCRIPTION
**Pull request by jitsedesmet**

### CODE REVIEW

File: packages/actor-query-operation-filter/lib/ActorQueryOperationFilter.ts [hunk 1]
```diff
@@ -1,46 +1,34 @@
...
-      const config = {
-        ...ActorQueryOperation.getAsyncExpressionContext(
-          context,
-          this.mediatorQueryOperation,
-          bindingsFactory,
-        ),
-      };
-      const _ = new AsyncEvaluator(dataFactory, operation.expression, config);
-      return passTestVoid();
+      const _ = await this.mediatorExpressionEvaluatorFactory.mediate({ algExpr: operation.expression, context });
```
Comment (rubensworks) [inline packages/actor-query-operation-filter/lib/ActorQueryOperationFilter.ts:26]: This can go, but perhaps later in a separate PR to `next/major`.

**Review by rubensworks — APPROVED** [review]
All looks good to me!

If the CI passes, and the last few commits have been squashed, I'll merge this.

**Comment by jitsedesmet** [comment]
I pray to every god for this to pass overnight and to be merged by morning.
I will not give my firstborn, but I will offer part of my mental stability. Payments were made in advance.

**Comment by rubensworks** [comment]
Your prayers have been answered, all looks good to me!
Very nice work @jitsedesmet, this has been a long journey, but the result is excellent!
This definitely adds a lot of value to Comunica.

**Pull request merged**